### PR TITLE
[release-2.2] fix hardcode namespace issue when getting acm route

### PR DESCRIPTION
**Pull request by yahliu**
Signed-off-by: yahliu <yahliu@redhat.com>

### CODE REVIEW

**Review by ChunxiAlexLuo — APPROVED** [review]
/hold

File: build/run-docker-tests.sh [hunk 1]
```diff
@@ -15,7 +15,8 @@ fi
 ./build/cluster-setup.sh
 
 #Get env from Docker arguments
-export SELENIUM_CLUSTER=https://`oc get route multicloud-console -n open-cluster-management -o=jsonpath='{.spec.host}'`
+installed_namespace=`oc get subscriptions.operators.coreos.com --all-namespaces | grep advanced-cluster-management | awk '{print $1}'`
```
Comment (ChunxiAlexLuo) [inline build/run-docker-tests.sh:18]: here bash syntax looks incorrect to me

File: build/run-e2e-tests.sh [hunk 2]
```diff
@@ -35,7 +35,8 @@ $DIR/cluster-setup.sh
 
 echo "Export envs to run e2e"
 export SERVICEACCT_TOKEN=`${BUILD_HARNESS_PATH}/vendor/oc whoami --show-token`
-export headerUrl=https://`oc get route multicloud-console -n open-cluster-management -o=jsonpath='{.spec.host}'`
+installed_namespace=`oc get subscriptions.operators.coreos.com --all-namespaces | grep advanced-cluster-management | awk '{print $1}'`
```
Comment (ChunxiAlexLuo) [inline build/run-e2e-tests.sh:38]: here bash syntax looks incorrect to me

**Review by ChunxiAlexLuo — CHANGES_REQUESTED** [review]

File: build/run-docker-tests.sh (hunk 1 above)
Comment (yahliu) [reply build/run-docker-tests.sh:18]: I can run it in local.   Can you please point which part is incorrect?   
```
root@yhliu-vm:~# bash -x ./a.sh
++ oc get subscriptions.operators.coreos.com --all-namespaces
++ grep advanced-cluster-management
++ awk '{print $1}'
+ installed_namespace=ocm
++ oc get route multicloud-console -n ocm '-o=jsonpath={.spec.host}'
+ export SELENIUM_CLUSTER=https://multicloud-console.apps.ocp4-gcp-2.gcp.dev09.red-chesterfield.com
+ SELENIUM_CLUSTER=https://multicloud-console.apps.ocp4-gcp-2.gcp.dev09.red-chesterfield.com
```

**Review by ChunxiAlexLuo — APPROVED** [review]
/lgtm

**Comment by yahliu** [comment]
@dhaiducek @ChunxiAlexLuo 2.2.4 already be released. can you help to merge this PR ?

**Comment by dhaiducek** [comment]
2.2.4 was prioritized for an urgent bug fix and we will probably need to wait for 2.2.5 to merge this. (We can check with Tony Fister to verify.)

**Pull request merged**